src/vstart.sh

```diff
@@ -360,12 +360,12 @@ fi
 ARGS="-c $conf_fn"
 
 prunb() {
-    echo "$* &"
+    printf "'%s' &" "$@"; echo;
```

Comment:
the output will not look right with the `&`,
```
'valgrind' &'--tool=memcheck' &'/var/ceph/ceph/build/bin/ceph-osd' &'-i' &'0' &'-c' &'/var/ceph/ceph/build/ceph.conf' &'-f' &
```
and what is the trailing `;` for?

might want to put
```sh
printf "'%s' " "$@"; echo '&'
```